x-pack/plugins/license_management/public/plugin.ts

```diff
@@ -76,8 +79,15 @@ export class LicenseManagementUIPlugin implements Plugin<void, void, any, any> {
         return renderApp(element, appDependencies);
       },
     });
+
+    return {
+      enabled: true,
+    };
   }
 
   start() {}
   stop() {}
 }
+
+export type LicenseManagementUIPluginSetup = ReturnType<LicenseManagementUIPlugin['setup']>;
```

Comment:
I would prefer to declare explicitly the setup interface above the class.
It makes it easier to scan IMO

```js
export interface LicenseManagementUIPluginSetup {
  enabled: boolean;
}

export class LicenseManagementUIPlugin ...
```

Reply:
Changes in 73e832b65c118e802e2e5bb1a85d834bee95f5af